app/controllers/stories_controller_decorator.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+ViewComponent::Storybook::StoriesController.class_eval do
+  before_filter :set_cors_policy
+
+  private
+
+  def set_cors_policy
```

Comment:
Great approach!

I'm just wondering if we should limit this to the development environment.

The extension can also be done in a cleaner way. You can define a new module with a method: `PermissiveCORSHeaders.before`. Then you can add this line in an initializer:
```rb
ViewComponent::Storybook::StoriesController.before_filter(PermissiveCORSHeaders) if Rails.env.development?
```

Examples:

- Look at `config/initializers/paypal.rb`.
- Search our code for `CurrentOrderLocker`.

Reply:
> I'm just wondering if we should limit this to the development environment.

Yes, I think as we know assume that StoryBook will only run for the development env. it's a great idea.

Reply:
There you go: b7399f43fe153efb71e36b38133691297d57ad0b !

Reply:
It's very strange, because I _sometimes_ had CORS error, and I need to restart the sever. Don't know why.

```
Access to fetch at 'http://localhost:3000/rails/stories/distributor_title_component/default?name=Freddy+s+Farm+Shop' from origin 'http://localhost:64504' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

Reply:
Okay, let's monitor that and see if it's a problem. Maybe there was just a caching problem on your machine.